autogen/coding/local_commandline_code_executor.py

```diff
@@ -128,7 +128,7 @@ def execute_code_blocks(self, code_blocks: List[CodeBlock]) -> CommandlineCodeRe
             )
             filename_uuid = uuid.uuid4().hex
             filename = None
-            if lang in ["bash", "shell", "sh"]:
+            if lang in ["bash", "shell", "sh", "pwsh"]:
```

Comment:
```suggestion
            if lang in ["bash", "shell", "sh", "pwsh", "powershell", "ps1"]:
```

Reply:
The `lang` here is only used for filename, which doesn't affect how it is executed. It uses `code_utils` later so the execution part is already handled.

Reply:
oh right okay